### ISSUE TITLE
Add "Create Desktop Shortcut" option from the GUI

### ISSUE THREAD

**Issue by aorin1** [Vita3K/Vita3K#1910]
The ability for creating shortcuts to start the games directly from the desktop or start menu, like RPCS3 would be a great addition to Vita3K.

![image](https://user-images.githubusercontent.com/32167620/176979838-b60051f9-2f32-488a-b454-5b6d6bd77808.png)

**Comment by EXtremeExploit** [comment]
This can already be done by right clicking the executable file > Send to > Desktop (Create Shortcut)

**Comment by Zangetsu38** [comment]
> This can already be done by right clicking the executable file > Send to > Desktop (Create Shortcut)

he want can get shurcout ready for run for any game ^^

**Comment by aorin1** [comment]
Yes, like the screenshot attached on the OP, RPCS3 allows users to easily create shortcuts for each game from its GUI, this is a very nice feature.

**Comment by NineKain** [comment]
Is this being considered?
